src/Datadog.Trace/TraceContext.cs

```diff
@@ -13,7 +13,7 @@ internal class TraceContext : ITraceContext
 
         private readonly DateTimeOffset _utcStart = DateTimeOffset.UtcNow;
         private readonly long _timestamp = Stopwatch.GetTimestamp();
-        private readonly List<Span> _spans = new List<Span>();
+        private List<Span> _spans = new List<Span>();
```

Comment:
Doesn't look like we needed to remove the `readonly`.

Reply:
Right, that because I started by doing something much more complicated 😅  gonna restore it